src/node_file.h

```diff
@@ -37,7 +37,7 @@ class FSContinuationData : public MemoryRetainer {
   }
 
   void PushPath(const std::string& path) {
-    paths.push_back(path);
+    paths.push_back(std::move(path));
```

Comment:
I don’t think this can work, because `path` is a const reference inside this function? If this function overload is currently unused, we can probably just remove it, though…